### PR TITLE
T361228 - Release WMDE.20

### PR DESCRIPTION
**Pull request by RickiJay-WMDE**
Release of WMDE.20

Removes the patch @rti made to fix the port visibility in 1.41.0

### CODE REVIEW

File: build/Wikibase/Dockerfile [hunk 1]
```diff
@@ -17,10 +17,6 @@ RUN git clone https://gerrit.wikimedia.org/r/mediawiki/extensions/Wikibase /tmp/
 FROM ${MEDIAWIKI_IMAGE_URL} as collector
 COPY --from=git-checkout /tmp/wikibase /var/www/html/extensions/Wikibase
 
-COPY ./fix-do-not-add-stdport-in-detectServer.patch /tmp/fix-do-not-add-stdport-in-detectServer.patch
```
Comment (lorenjohnson) [inline build/Wikibase/Dockerfile:20]: What was this for previously?

Comment (RickiJay-WMDE) [reply build/Wikibase/Dockerfile:20]: Applying Robert's patch, which fixed the port number bug. WebRequest.php has been updated such that the patch now causes build to fail, and the issue it was patching has been resolved

Comment (lorenjohnson) [reply build/Wikibase/Dockerfile:20]: Got it. So this was a new patch with 1.41.0, now fixed in 1.41.1

File: CHANGES.md [hunk 2]
```diff
@@ -2,6 +2,16 @@
...
 
+## April 2024: Security releases for 1.41.1
+
+- [MediaWiki release announcement: 1.41.1](https://lists.wikimedia.org/hyperkitty/list/wikitech-l@lists.wikimedia.org/thread/V3WXEPXV2DU6WTVEKK4XHW4QXD5OFKD7/)
+- [MediaWiki full release notes](https://www.mediawiki.org/wiki/Release_notes/1.41#MediaWiki_1.41.1)
+- [Wikibase Suite Release task T361228](https://phabricator.wikimedia.org/T361228)
+
+| Suite Version | Version Info | Date available | Release task |
+| --- | --- | --- | --- |
+| wmde.20 ([spec](https://github.com/wmde/wikibase-release-pipeline/blob/wmde.20/variables.env)) | MediaWiki (1.41.1, PHP 8.2.15, debian/apache 2.4.57-2), WDQS (0.3.137) | XX April 2024 | [T361228](https://phabricator.wikimedia.org/T361228) |
```
Comment (lorenjohnson) [inline CHANGES.md:13]: Re. `XX April 2024`: Understanding that we don't know the value until we actually merge, but this should be populdated. The wide.17 note below also is `XX March 2024` and should be updated to the actual release date.

Comment (RickiJay-WMDE) [reply CHANGES.md]: Corrected `XX March`, guessed on `XX April`

**Review by lorenjohnson — APPROVED** [review]
LGTM! 🎉

**Review by rti — APPROVED** [review]

**Pull request merged**